specs/0113-light-theme.md

```diff
@@ -0,0 +1,35 @@
+- Feature Name: Light theme
+- Created: 2023-06-22
+- Associated Issue: #2141
+
+# Summary
+
+Implement light theme and enable users to switch between the (current) dark theme and the light theme.
+
+# Motivation
+
+Over the years, many users have expressed preference for a light color theme over the Pontoon default dark theme. Having the ability to switch themes will also allow us to implement high contrast themes, which benefit low vision users, increase readability and reduce website’s visual noise.
+
+# Implementation notes
+
+In the first step, we'll replace existing hardcoded color values with CSS variables. We'll store those in separate CSS files, with one version of each file for each theme. Users will be able to switch theme in their settings.
```

Comment:
Should there be a third option to follow browser/OS settings? Dark, Light, Automatic.

I also think the specs should be a bit more prescriptive on where this setting would be, and what it would say. Without it, we're deferring the decision to the implementation phase.

Reply:
Ultimately, the automatic option should be the default one. Maybe not initially though, to let us effectively beta-test the light theme with those who actively switch to it.